src/Drupal/Commands/sql/SanitizeUserFieldsCommands.php

```diff
@@ -66,7 +66,7 @@ public function sanitize($result, CommandData $commandData): void
             $query = $conn->update($table);
             $name = $def->getName();
             $field_type_class = \Drupal::service('plugin.manager.field.field_type')->getPluginClass($def->getType());
-            $supported_field_types = ['email', 'string', 'string_long', 'telephone', 'text', 'text_long', 'text_with_summary'];
+            $supported_field_types = ['email', 'string', 'string_long', 'telephone', 'text', 'text_long', 'text_with_summary', 'integer', 'decimal'];
             if (in_array($def->getType(), $supported_field_types)) {
                 $value_array = $field_type_class::generateSampleValue($def);
                 $value = $value_array['value'];
```

Comment:
This should generate a value that respects max/min and such. No need to hard code a value.